docs/usage.rst

```diff
@@ -221,6 +221,10 @@ Methods for analysing import chains
     :param str importer: The module at the start of a potential chain of imports between ``importer`` and ``imported``
         (i.e. the module that potentially imports ``imported``, even indirectly).
     :param str imported: The module at the end of the potential chain of imports.
+    :param bool as_packages: Whether to treat the supplied modules as individual modules,
```

Comment:
We should add this to the signature on l. 219 too. (While we're at it, would you mind adding it to the signature of `find_shortest_chains` on l. 231, too? I missed that one.)